src/test/suite/fixtures/recorded/compoundTargets/postBatVerticalUntilFine.yml

```diff
@@ -44,10 +44,12 @@ finalState:
     - anchor: {line: 3, character: 1}
       active: {line: 3, character: 1}
   thatMark:
-    - contentRange:
+    - type: PlainTarget
```

Comment:
vertical ranges return plain targets, and "post" leverages `setSelection`, so it now preserves this information